docs/quickstart.md

```diff
@@ -0,0 +1,84 @@
+Tilt Installation Guide
+=======================
+
+Tilt is currently available for MacOS and Linux.
+
+You'll also need:
+
+- Docker, to build containers
+- Kubectl, to cuddle your cluster
+- A local Kubernetes cluster (on MacOS, Docker For Mac works for this!)
```

Comment:
Do we not want to advertise running on GKE?

Reply:
i think we probably shouldn't for the initial install? setting up GKE is non-trivial, and might not even be possible if you don't have access to your team's cluster